Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -8,12 +12,15 @@ namespace Amazon.Ion.ObjectMapper.Test
     [TestClass]
     public class IonObjectSerializerTest
     {
+        IValueFactory valueFactory = new ValueFactory();
+        Truck nativeTruck = new Truck();
```

Comment:
These sort of objects are defined in `TestObjects.cs`

Reply:
Moved.

Reply:
`nativeTruck` is still here.